### PR TITLE
ci: Run live tests in circleci

### PR DESCRIPTION
**Pull request by KitBurgess**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,11 +29,17 @@ jobs:
             - ~/.cache/pip
 
       - run:
-          name: "Run tests"
+          name: "Run local tests only "
```
Comment (kujon) [inline .circleci/config.yml]: Local here refers to the mocked api?

File: .circleci/config.yml [hunk 2]
```diff
@@ -29,11 +29,17 @@ jobs:
...
           command: |
             . .venv/bin/activate
             export SKIP_TAGS=real; nose2
 
+      - run:
+          name: "Run all tests"
+          command: |
+            . .venv/bin/activate
+            nose2
```
Comment (kujon) [inline .circleci/config.yml]: Does this mean that the CI is running the local tests twice?

File: .circleci/config.yml (hunk 1 above)
Comment (KitBurgess) [reply .circleci/config.yml]: Yes

File: .circleci/config.yml (hunk 2 above)
Comment (KitBurgess) [reply .circleci/config.yml]: Yes it does

File: .circleci/config.yml (hunk 1 above)
Comment (KitBurgess) [reply .circleci/config.yml]:
```suggestion
          name: "Run local tests against mocked api"
```

File: .circleci/config.yml (hunk 2 above)
Comment (kujon) [reply .circleci/config.yml]: Shall we not do that and have shorter build times?

Comment (KitBurgess) [reply .circleci/config.yml]: We could do, the local tests currently take <1second to run, so i don't think it's a problem right now.

**Comment by KitBurgess** [comment]
Fat finger merged that prematurely by accident

**Pull request merged**